recipe/build.sh

```diff
@@ -21,4 +21,5 @@ fi
 ./configure --with-pic --prefix=$PREFIX --disable-dependency-tracking
 make
 make check
+ln README.md README  # hack; snappy autotools build is broken in 1.1.6
```

Comment:
This is a hard link, so it saves ten bytes or whatever on the file system. :)  Copy would be the same, doesn't matter.